packages/cli/src/commands/setup/i18n/i18n.js

```diff
@@ -169,6 +170,17 @@ export const handler = async ({ force }) => {
         }
       },
     },
+    {
+      title: 'Configure Storybook...',
```

Comment:
```suggestion
      title: 'Configuring Storybook...',
```
It's `Configuring i18n...` earlier in the file, so let's make it "Configuring" here as well